stories/components/agenda/agenda.style.js

```diff
@@ -14,17 +14,8 @@ export default {
   decorators: [withCodeEditor]
 };
 
-export const darkTheme = () => html`
-  <mgt-agenda class="mgt-dark"></mgt-agenda>
-  <style>
-    body {
-      background-color: black;
-    }
-  </style>
-`;
-
-export const customCSSProperties = () => html`
-  <mgt-agenda class="agenda" group-by-day></mgt-agenda>
+export const customProperties = () => html`
```

Comment:
We need to use the name of customCSSProperties to ensure that our convention based exclusion of the theme toggle in Storybook continues to work